package.json

```diff
@@ -40,11 +42,13 @@
     "react-router-dom": "^4.3.1",
     "react-showdown": "^1.6.0",
     "react-sidebar": "^3.0.2",
+    "react-tippy": "^1.3.1",
     "react-toastify": "^4.5.2",
     "reactstrap": "^8.2.0",
     "secp256k1": "^3.8.0",
     "styled-components": "^4.4.1",
     "tendermint": "^4.0.8",
+    "tippy.js": "^5.1.4",
```

Comment:
Did you use this `tippy.js`? I don't see that in the code. `imports` should already been added in dependencies with `Meteor`. Did you addd it again?

Reply:
I was planning to use tippy.js at the beginning but then I decided to use Popover instead.

Reply:
This PR is working fine. Please remove the unused packages and it will be merged. What you can do is to run
`meteor npm uninstall <package-name>`. Meteor will manage how it should organize with the package.json.